### PR TITLE
feat: Check if Account Name exists

### PR DESCRIPTION
**Pull request by iuricmp**
This PR implements a React Native [Modal](https://reactnative.dev/docs/modal) to confirm if the user wants to override an existing account. 

Fix for #30

### CODE REVIEW

File: gnoboard/src/hooks/use-gno.ts [hunk 1]
```diff
@@ -89,6 +82,24 @@ export const useGno = (): GnoResponse => {
     return response.phrase;
   };
 
+  const isAccountNameExists = async (nameOrBech32: string) => {
```
Comment (iuricmp) [inline gnoboard/src/hooks/use-gno.ts:85]: Utility function to simplify the UI process.

File: gnoboard/src/screens/certify/create-password/index.tsx [hunk 2]
```diff
@@ -38,36 +41,55 @@ const CreatePassword: React.FC<Props> = ({ route }) => {
     }
 
     try {
+      if (!override) {
+        const existingAccountName = await gno.isAccountNameExists(name);
```
Comment (jefft0) [inline gnoboard/src/screens/certify/create-password/index.tsx:45]: If you rebase to the latest main branch, you can use gno.hasKeyByName.
https://github.com/gnolang/gnomobile/blob/9566241f50aea08c0611d410c5aeeb27ba3821a6/gnoboard/src/hooks/use-gno.ts#L93

File: gnoboard/src/hooks/use-gno.ts
Comment (D4ryl00) [inline gnoboard/src/hooks/use-gno.ts:1]: Why did you delete hasKeyByName (and other has... functions)?

File: gnoboard/src/screens/certify/create-password/index.tsx [hunk 3]
```diff
@@ -38,36 +41,55 @@ const CreatePassword: React.FC<Props> = ({ route }) => {
     }
 
     try {
+      if (!override) {
+        const existingAccountName = await gno.getKeyInfoByName(name);
```
Comment (D4ryl00) [inline gnoboard/src/screens/certify/create-password/index.tsx:45]: Why don't you use hasKeyByName here to test if there is an account or not?

File: gnoboard/src/hooks/use-gno.ts
Comment (iuricmp) [reply gnoboard/src/hooks/use-gno.ts:1]: Fixed. A bad rebase occurred.

**Review by D4ryl00 — APPROVED** [review]
LGTM 👍
